### ISSUE TITLE
[COGNITION]: Please provide a Submit button for the Outreach materials view, and possibly a reset button

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#901]
## Status
**[2024-07-15] [Fran]** Added to our Product Sync list for Michelle to review/comment on disposition.

## Description

We are currently reloading the Outreach materials view when users make a selection from one or both `<select>` menus. This can break users' expectation of the menu's intended action. This can be improved by adding a Submit button, and possibly a Reset button to restore all examples. Screenshot attached below.

Tagging @smoonpeyes for a design consult on placement and styling of the additional controls.

## Point of Contact

**VFS Point of Contact:** _Trevor, Layla_

## Acceptance Criteria

* As a user, I want to be able to make selections from one or both of the `<select>` menus, then press a submit button to filter results
* As a user, I would appreciate a Reset button to restore all items to the view below

## Environment
* MacOS Mojave
* Chrome latest

## WCAG or Vendor Guidance (optional)
* [On Input: Understanding SC 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-09 at 3.59.33 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/4c943c5f-57e4-479f-8aa0-5b5ce998d40a)

**Comment by jefflbrauer** [comment]
@andrewlewandowski I will add this ticket to our backlog so we can groom together. We will need an updated design.

**Comment by andrewlewandowski** [comment]
@1Copenut - Thanks for flagging this. Can you please connect directly with @rtwell on the 508 issue, and possible design solutions? If we need to significantly update the UI design, we will need to add it to the CMS team's backlog (@jefflbrauer ).

**Comment by 1Copenut** [comment]
Adding @rtwell as an assignee for visibility.

**Comment by rtwell** [comment]
My proposal: add a clear call to action button, and also add a H2 that describes the functionality of the filters. @jenniferlee-dsva i look to you for the correct words for the button and h2. 

This also means we have to disable the dynamic filtering that triggers as soon as a user selects a topic or file type. Now, a user would select an item from "topics" or "file type" (or both) and click the Filter button and then the content below would filter. 

thoughts?

![image](https://user-images.githubusercontent.com/12564977/63364518-bcd7df80-c343-11e9-8cd4-b2c6a993b7b1.png)

**Comment by jenniferlee-dsva** [comment]
^ @rtwell (or @andrewlewandowski ) - 2 things:

1/ re functionality: does the user need to filter by both or either? Also, what's in the topic dropdown -- our benefit categories or other topics, too? 

2/ in the sample above, we're using "benefit materials" and "outreach materials" interchangeably. Also we're incorrectly referring to VA with "the" in front of it. 

Lmk answers to #1 and I can better provide a few quick copy edits.

**Comment by rtwell** [comment]
@jenniferlee-dsva you can see the live build here: https://staging.va.gov/outreach-and-events/outreach-materials/

Users can filter one or the other, or both.

**Comment by jenniferlee-dsva** [comment]
Thanks @rtwell 

1/ 
Filter by benefit or file type
Select a benefit topic 
- All benefit topics - 

Select a file type
- All file types - 

2/ ^ in the file types dropdown, there's a small  mistake (capitalization and missing serial comma. 
"Poster, Flyer, brochure and fact sheets" should be "Poster, flyer, brochure, and fact sheets"  

3/ A few other mistakes on the rest of this page. @andrewlewandowski - should I drop them here or do you want to open another ticket or just Slack Ryan Sibley (don't know who was content person for Office/outreach)?

**Comment by andrewlewandowski** [comment]
@RyanSibley - Please add a ticket to the CMS team backlog to address the above content items. Thanks!
Cc: @jefflbrauer

**Comment by RyanSibley** [comment]
Here's the issue: https://va-gov.atlassian.net/browse/VAGOV-5650

**Comment by rtwell** [comment]
comps updated:
![image](https://user-images.githubusercontent.com/12564977/63547989-539acc80-c4fb-11e9-9810-df58b3844cd0.png)

**Comment by 1Copenut** [comment]
@rtwell I'm digging this new comp. Wanted to ask if the **Viewing 1-10 of N results** text was removed intentionally, or if we could bring it or some other element back to indicate how filters have affected the page results. I'm thinking ahead a bit for assistive devices needing a text node to hook onto and announce the change.

LMK if you want to sync up on this in a call or chat.

**Comment by rtwell** [comment]
good catch @1Copenut — I think I left that out, mistakenly.

**Comment by FranECross** [comment]
@laflannery Can we chat about this item in our meeting today? I'm unclear whether it was going to go to the CMS team, or if it's something we should address. It's the only ticket left to get done in an older Epic. Thanks!

**Comment by laflannery** [comment]
@FranECross I had thought that outreach materials was going way and being moved somewhere else, which is why ticket I have pertaining to Outreach were Iceboxed. Maybe check with Jill/Dave C to see if that's still the case?

**Comment by jilladams** [comment]
@FranECross yolo-ing and closing this. You're right that Outreach Materials are in maintenance mode until they can be deprecated by a replacement product. We can always reopen later as needed.
